eng/Versions.props

```diff
@@ -29,7 +29,11 @@
   <PropertyGroup>
     <RestoreSources>
       $(RestoreSources);
-      https://dotnetfeed.blob.core.windows.net/dotnet-core/packages/index.json
+      https://dotnet.myget.org/F/nuget-build/api/v3/index.json;
```

Comment:
Do we need all of these?

Reply:
Just to be in sync with what's in CLI. I don't really sure for this specific repo

Reply:
I merged it first. to get CLI building